prov/rxm/src/rxm.h

```diff
@@ -321,6 +322,26 @@ extern struct fi_domain_attr rxm_domain_attr;
 extern struct fi_tx_attr rxm_tx_attr;
 extern struct fi_rx_attr rxm_rx_attr;
 
+static inline void
+rxm_copy_to_iov(const struct iovec *iov, size_t iov_count,
+		void *buf, uint64_t bufsize)
+{
+	if (iov_count > 1)
+		ofi_copy_to_iov(iov, iov_count, 0, buf, bufsize);
+	else
+		memcpy(iov[0].iov_base, buf, bufsize);
```

Comment:
Let's move the memcpy directly into the ofi_copy_*_iov routines, except let's replace the memcpy with a direct assignment, since we're talking about 2 fields.

Reply:
For some dumb reason I started thinking that we were copying the iov itself.  I need more sleep.  Thanks - waiting for CI to finish then merging.

Reply:
@shefty Thanks for reviewing of my patch :)